ethereum/core/src/jmh/java/org/hyperledger/besu/ethereum/vm/operations/Keccak256Benchmark.java

```diff
@@ -21,15 +21,24 @@
 
 import org.apache.tuweni.bytes.Bytes;
 import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OperationsPerInvocation;
 import org.openjdk.jmh.annotations.OutputTimeUnit;
 import org.openjdk.jmh.annotations.Param;
 import org.openjdk.jmh.annotations.Scope;
 import org.openjdk.jmh.annotations.Setup;
 import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
 
 @State(Scope.Thread)
-@OutputTimeUnit(value = TimeUnit.MILLISECONDS)
-public class Hashing20kKeccak256Benchmark {
+@Warmup(iterations = 1)
+@OutputTimeUnit(value = TimeUnit.NANOSECONDS)
+@Fork(2)
+@BenchmarkMode(Mode.AverageTime)
+public class Keccak256Benchmark {
+  private static final int OPERATIONS_PER_INVOCATION = 20_000;
```

Comment:
Why is this different to CLZ?

Reply:
Feel like we could just adjust the JMH settings like we're doing already, rather than introduce another variable? Unless there's some domain-specific benefit to having the inner loop?